thunderbirdparser/src/org/sleuthkit/autopsy/thunderbirdparser/PstParser.java

```diff
@@ -237,7 +245,7 @@ private void extractAttachments(EmailMessage email, PSTMessage msg, long fileID)
                 attachment.setSize(attach.getFilesize());
                 attachment.setEncodingType(TskData.EncodingType.XOR1);
                 email.addAttachment(attachment);
-            } catch (PSTException | IOException | NullPointerException ex) {
+            } catch (PSTException | IOException | NullPointerException | NoCurrentCaseException ex) {
```

Comment:
The NoCurrentCaseException should be in a separate catch so the error can be logged as SEVERE.